docs/remote/troubleshooting.md

```diff
@@ -163,6 +163,10 @@ If you encounter an error when connecting, you may need to enable socket forward
 
 If you are running into problems with VS Code hanging while trying to connect (and potentially timing out), there are a few things you can do to try to resolve the issue.
 
+**Server fails to start**
+
+If you get a message along the lines of "Could not establish connection to `server_name`: The VS Code Server failed to start," try running the command: `Remote-SSH: Kill VS Code Server on Host`. This will remove the server and can help in scenarios such as it was installed in a bad state if the host ran out of disk space.
```

Comment:
Running this command can fix a lot of issues, it's a good general troubleshooting tip

Reply:
I see this command helps a lot of folks in remote-release, so I figured it'd make sense to call out in some higher level section in this doc. 

I've reworded this section to be a bit more of a general troubleshooting tip (i.e. now titled "General troubleshooting: Remove the server," updated the explanation). 

If you have any other feedback on the wording or placement (i.e. I felt like under the `Troubleshooting hanging or failing connections` made sense, but open to feedback here), please let me know.